ChangeLog

```diff
@@ -32,6 +32,13 @@ cdeps
 
 
 cice
+    Chris Fischer 2025-09-25 - cesm3_cice6_6_1_5 - components/cice (cesm3_0_alpha07e)
+    https://github.com/ESCOMP/CESM_CICE/tags/cesm3_cice6_6_1_5
```

Comment:
above you have 6_6_1_4

Reply:
I think there's an issue with 6_6_1_5, I was trying 6_6_1_4 as a test.